CONTRIBUTING.md

```diff
@@ -43,13 +43,17 @@ When the [ECMA-402 task group](https://www.ecma-international.org/task-groups/tc
 
 The ECMA-402 task group will also review proposals to *improve existing features*.  We will consider such proposals so long as the champion can demonstrate that their request will not make an ECMA-402 implementation substantially more complex.  If the request requires a nontrivial increase in locale data size, we will hold the request against the criteria for new features listed above.
 
+In addition to these requirements, a proposal must have an associated request for a [W3C i18n Review](https://www.w3.org/International/review-request) opened at least 3 weeks prior to the TG2 meeting at which it is seeking approval for Stage 2.
+
 ### ECMA-402 Stage 3 Requirements
 
 The ECMA-402 task group does not want to substantially raise the bar for current and future implementations. In order for a proposal to be accepted as Stage 3, the proposal champion must demonstrate:
 
 1. **Payload Mitigation**
     1. Not all proposals are equal; some may require large amounts of locale data. The proposal champion must verify with browser vendors that the proposal meets their standard for payload size increase. The proposal may need to be modified to reduce payload size increases if requested by browser vendors.
 
+If the proposal has seen substantive changes since Stage 2, the aforementioned W3C i18n Review should also be updated to reflect those changes.
```

Comment:
Probably this should say "request a new review"? Multiple reviews of the same document are common and low-cost.